dist/profile/templates/confluence/vhost.conf

```
@@ -2857,6 +2857,12 @@ RewriteRule "^/display/JENKINS/App\+Center\+Plugin$" "https://plugins.jenkins.io
 RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
 RewriteRule "^/display/JENKINS/XML\+Job\+to\+Job\+DSL$" "https://plugins.jenkins.io/xml-job-to-job-dsl/" [NC,L,QSA,R=301]
 RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Task\+Scanner\Plugin" "https://plugins.jenkins.io/warnings-ng//" [NC,L,QSA,R=301]
```

Missing the `+` character that separates the word "Scanner" from the word "Plugin" in the wiki URL.  Detected by the tests in #1594, though the message when detected is not clear about the cause, just that the page `/display/JENKINS/Task+ScannerPlugin` does not exist.
```suggestion
RewriteRule "^/display/JENKINS/Task\+Scanner\+Plugin" "https://plugins.jenkins.io/warnings-ng//" [NC,L,QSA,R=301]
```